controllers/storageoscluster/controller.go

```diff
@@ -98,6 +99,13 @@ func (c *StorageOSClusterController) UpdateStatus(ctx context.Context, obj clien
 	apiManagerCondition := getAPIManagerCondition(ctx, c.Client, obj.GetNamespace(), log)
 	meta.SetStatusCondition(&cluster.Status.Conditions, apiManagerCondition)
 
+	if cluster.Spec.EnablePortalManager {
+		portalManagerCondition := getPortalManagerCondition(ctx, c.Client, obj.GetNamespace(), log)
+		meta.SetStatusCondition(&cluster.Status.Conditions, portalManagerCondition)
+	} else {
+		meta.RemoveStatusCondition(&cluster.Status.Conditions, portalManagerReadyType)
```

Comment:
Out of interest what does removing the status condition do?

Does it then show as unready?

Reply:
No, it removes the given entry.